OneCommunityGlobal/HighestGoodNetworkApp
# Wanting add copy icon to weekly summary report

# Description
<img width="887" alt="copy icon task" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105323719/7837821e-4c59-4876-bd51-663fb5553d1a">


## Related PRS (if any):
…

## Main changes explained:
- Updated file FormattedReport.jsx to add copy icon in 'weeklySummaryReport' page (the copy icon will be hided when the summary is either "not provided" or "not required for this user")

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log in as any user who has permission to see "weeklySummaryReport" (Recommend: Owner) 
4. submit a weekly summary  
5. go to tab "Report" -> Weekly Summaries Report -> find the user who has a valid summary content (the tab is set to "last week" by default, remember to switch to "this week")
6. check there is a copy icon at the top-right corner of the weekly summary and the copy-to-clipboard function works well


## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105323719/65d2709e-528e-4a41-81ce-ca4d9708a4ca


## Note:
When the summary is not provided or not required for the user, should not show the copy icon.
